apps/mobile/android/app/build.gradle

```diff
@@ -99,12 +99,17 @@ android {
             keyPassword 'android'
         }
         release {
-            if (project.hasProperty('LEATHER_UPLOAD_STORE_FILE')) {
-                storeFile file(LEATHER_UPLOAD_STORE_FILE)
-                storePassword LEATHER_UPLOAD_STORE_PASSWORD
-                keyAlias LEATHER_UPLOAD_KEY_ALIAS
-                keyPassword LEATHER_UPLOAD_KEY_PASSWORD
-            }
+
+            storeFile file('debug.keystore')
+            storePassword 'android'
+            keyAlias 'androiddebugkey'
+            keyPassword 'android'
+            // if (project.hasProperty('LEATHER_UPLOAD_STORE_FILE')) {
```

Comment:
@camerow on the hackathon, it wouldn't work until I disabled `if (project.hasProperty('LEATHER_UPLOAD_STORE_FILE')` then I just copied in the `debug.keystore` etc. to get it working locally but I think thats not correct (although working)